capabilities/src/execute_task_solution_capability.cpp

```diff
@@ -94,9 +94,9 @@ void ExecuteTaskSolutionCapability::initialize() {
 	                                             std::placeholders::_1, std::placeholders::_2)),
 	    ActionServerType::CancelCallback(
 	        std::bind(&ExecuteTaskSolutionCapability::preemptCallback, this, std::placeholders::_1)),
-	    [this](const std::shared_ptr<rclcpp_action::ServerGoalHandle<ExecuteTaskSolutionAction>>& goal_handle) {
-		    last_goal_future_ =
-		        std::async(std::launch::async, &ExecuteTaskSolutionCapability::goalCallback, this, goal_handle);
+	    [this](std::shared_ptr<rclcpp_action::ServerGoalHandle<ExecuteTaskSolutionAction>> goal_handle) {
+		    last_goal_future_ = std::async(std::launch::async, &ExecuteTaskSolutionCapability::goalCallback, this,
+		                                   std::move(goal_handle));
```

Comment:
Are these changes still necessary after Abi fixed the clang-tidy failure differently on the `ros2` branch https://github.com/PickNikRobotics/moveit_task_constructor/commit/d202984e845235368c2a2ee961d66434872fd14f
```suggestion
	    [this](const std::shared_ptr<rclcpp_action::ServerGoalHandle<ExecuteTaskSolutionAction>>& goal_handle) {
		    last_goal_future_ =
		        std::async(std::launch::async, &ExecuteTaskSolutionCapability::goalCallback, this, goal_handle);
```